README.md

```diff
@@ -100,7 +100,8 @@ As the plugin is shell-powered, one can easily run destructive commands just by
 -   Long-running commands can cause issues. E.g. if you type `{{shell:sleep 10 && echo 1}}`<kbd>Tab</kbd> and before it finishes type `{{now}}`<kbd>Tab</kbd>, then `{{now}}` will be replaced with `1`.
 -   Windows support is currently limited:
     -   Shortcuts with `replacement` field are processed correctly
-    -   To use `command` field on Windows, you need to set the `Settings > Plugin Options > Text Expander > Shell executable` to `powershell` and use Powershell syntax in your commands.
+    -   To use `command` field on Windows you need to have a [WSL Subsystem](https://docs.microsoft.com/en-us/windows/wsl/install-win10) installed and set `Settings > Plugin Options > Text Expander > Shell executable` to `your os name.exe`(`ubuntu.exe` for Ubuntu)(also make sure any python packages you use are installed)

```

Comment:
Everything is fine, except for the following:
1. `<your os name>.exe` would be more conventional
2. ` Ubuntu)(also` -- these kissing parentheses look weird, comma would be more appropriate:  ` Ubuntu), also`